src/models/participant/facade.js

```diff
@@ -0,0 +1,60 @@
+/*****
```

Comment:
Same here should also be moved to a common lib.

Reply:
I agree - let's cover it as a task in https://github.com/mojaloop/project/issues/848